src/styles/components/_home-search.scss

```diff
@@ -93,5 +95,143 @@
       left: 50%;
       transform: translate(-50%, -50%);
     }
+
```

Comment:
All these codes should be either in `home-search-box` of `home-search-box-suggestion` not `home-search` 🥲